data/json/professions.json

```diff
@@ -3417,5 +3417,460 @@
+        "skills": [
+            {
+                "level": 1,
+                "name": "dodge"
+            },
+            {
+                "level": 1,
+                "name": "throw"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+                "jacket_light",
+                "jeans",
+                "socks",
+                "lowtops",
+                "fanny",
+                "juice"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "boy_shorts"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "science_club_mem",
+        "name": "Science Club Member",
+        "description": "You were a member of the school science club, and right now you're as upset as you've ever been that the school wouldn't let you play with the really fun chemicals that make things go boom. At least now no one's around to tell you that you can't.",
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "cooking"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+                "jacket_light",
+                "jeans",
+                "socks",
+                "sneakers",
+                "knit_scarf",
+                "glasses_safety",
+                "chemistry_set",
+                "textbook_chemistry"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "av_club_mem",
+        "name": "A/V Club Member",
+        "description": "You were a member of the school A/V club. You're sure there's some way you can use your technical skills to help stay alive. You just haven't figured out how to make an awesome death ray yet.",
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "electronics"
+            },
+            {
+                "level": 1,
+                "name": "computer"
+            }
+        ],
+        "items": {
+            "both": [
+                "linuxtshirt",
+                "jacket_light",
+                "jeans",
+                "socks",
+                "sneakers",
+                "knit_scarf",
+                "backpack",
+                "mag_electronics"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "janitor",
+        "name": "Janitor",
+        "description": "You earned a living from sweeping up chocolate wrappers and picking chewing gum from under tables. Now the only thing you'll be sweeping are the brains of the dead.",
+        "points": -1,
```

Comment:
No significant* carrying capacity, lightweight clothing, and no knife/fire.  This one might qualify for -2.

*As in, I'm thinking you'd have to wield the mop in order to carry anything else effectively.  Same problem the various athletic folks have.